Library/Formula/rocksdb.rb

```diff
@@ -20,6 +18,9 @@ class Rocksdb < Formula
   depends_on "snappy"
   depends_on "lz4"
 
+  # fix the weird library naming scheme in rocksdb Makefile
+  patch :DATA
```

Comment:
On Fri, 31 Jul 2015, at 04:56 PM, Dominyk Tiller wrote:
> > @@ -20,6 +18,9 @@ class Rocksdb < Formula
> >    depends_on "snappy"
> >    depends_on "lz4"
> >  
> > +  # fix the weird library naming scheme in rocksdb Makefile
> > +  patch :DATA
> 
> Doesn't seem to be done?

It has been updated, yes. The :DATA section matches
https://patch-diff.githubusercontent.com/raw/facebook/rocksdb/pull/670.patch
if you look closely. It's not that different from my original patch
though. The main difference is platform detection & commit details.